.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    strategy:
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install package dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install pyyaml requests
+          for pathfile in $(find ./ -name "requirements*.txt" -type f); do
+             pip install -r $pathfile
+          done
+      # pytest
+      - name: Install test dependencies
+        run: |
+          python -m pip install pytest pytest-mock pytest-custom_exit_code pytest-cov
```

Comment:
OK. I'd like to introduce requirements-dev.txt in the tests directory. It's OK, not this PR.